ekncontent/configure.ac

```diff
@@ -76,6 +76,13 @@ AC_DEFINE([EKN_CONTENT_VERSION], [_EKN_CONTENT_VERSION_MACRO], [Full version num
 #      release, then set age to 0.
 AC_SUBST([EKN_CONTENT_LT_VERSION_INFO],[0:0:0])
 
+# We want to log queries for debugging in development versions, but not in
+# released versions since they can contain sensitive information.
+AX_IS_RELEASE([minor-version])
```

Comment:
I think relying on `G_MESSAGES_DEBUG` is okay. I didn't realize that the messages are not logged to the journal if the environment variable isn't present. And it would be difficult to set that environment variable unintentionally, especially when running in a sandbox.